js/test/name/testnamefmt.js

```diff
@@ -745,9 +746,25 @@ module.exports.testnamefmt = {
             locale: "ko-KR"
         });
 
-        test.equal(fmt.format(name), "닥터 박은성");
+        test.equal(fmt.format(name), "박은성 님");
         test.done();
     },
+    /*testNameFmtKOFamiliar2: function(test) {
+        test.expect(1);
+        var name = new Name({
+            givenName: "은성",
+            familyName: "박"
+        }, {
+            locale: "ko-KR"
+        });
+        var fmt = new NameFmt({
+            style: "familiar",
+            locale: "ko-KR"
+        });
+
+        test.equal(fmt.format(name), "박은성 님");
+        test.done();
+    },*/
```

Comment:
@ehoogerbeets What do you think about the set default value even developer don't put suffix? (This testcase doesn't activate at this point)

Reply:
We can do that in a separate PR